resources/definitions/HardPrint_Hozo.def.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": 2,
+    "name": "HardPrint Hozo, Hozo",
```

Comment:
I was unable to find any information about this printer. Is it being sold anywhere? Could you provide a link to a sales page?

Reply:
Hello, we are Hardprint Co, an entrepreneurial team from Taiwan, currently stationed in Taiwan’s largest Maker Space M.zone.
Hozo, Hozo and Hozo iX are our DIY educational 3D printers, which are expected to be promoted and sold in February. Below is a link to our promotional video.

https://fb.watch/anBOWpgUwC/

Thank you very much